databases.go

```diff
@@ -659,6 +661,27 @@ type MongoDBConfig struct {
 	Verbosity                       *int    `json:"verbosity,omitempty"`
 }
 
+// KafkaConfig holds advanced configurations for Kafka database clusters.
+type KafkaConfig struct {
```

Comment:
From the conversation yesterday, sounds like we want `*int64` for at least some of these, right?

Reply:
Updated in [c8e9ba3](https://github.com/digitalocean/godo/pull/727/commits/c8e9ba3aef72c1ea6cc0e90209976c182fcc8c20). 
I'm using `math/big` because `int64` is not enough for some values.
API doc for reference: https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_get_config